### PR TITLE
feat(html,css,js-dom): curr updates: no bootstrap, more css & html practice

### PR DESCRIPTION
**Pull request by alodahl**
(In progress): Updates for H2 discussed between @SarahCawley and I based on current trends & participant feedback:
- no longer require bootstrap
- slow down & break up css outline
- more basic css practice
- more basic html practice
- don't require CLI or VSCode until after HTML & CSS intros

### CODE REVIEW

File: projects/recipe-page/phase-2-css-prompt.md [hunk 1]
```diff
@@ -65,14 +66,12 @@ You'll be building on the HTML code you completed in Phase 1 of this project.
...
-2. Use the Terminal shortcut `code .` to open the `recipe-page` directory in VSCode.
-3. Run `git status`. You should see that 'styles.css' is not being tracked yet.
-4. Add and commit, making sure to write a descriptive commit message in the present tense, such as "Create stylesheet."
+1. On your desktop, create a folder called `techtonica-projects`.  Move your `recipe-page.html` file into `techtonica-projects`.  If you no longer have it, make it again with the Challenge directions from [Phase 1](./phase-1-html-prompt.md).
+2. Use  TextEdit or WordPad to make a new file called `styles.css`. Make sure it is in the `techtonica-projects` directory right next to `recipe-page.html`.
 
 #### Part 2 - Link to your CSS file from within your HTML `<head>` tags
 
-1. Find the `styles.css` file in VSCode that you just created. Add this small bit of CSS code to the file, then save the file.
+1. In `styles.css`, add this small bit of CSS code to the file, then save the file.
```
Comment (SarahCawley) [inline projects/recipe-page/phase-2-css-prompt.md:74]: Is there a reason we took out the vscode?

File: projects/recipe-page/phase-2-css-prompt.md [hunk 2]
```diff
@@ -65,14 +66,12 @@ You'll be building on the HTML code you completed in Phase 1 of this project.
 
 #### Part 1 - Create a CSS file in the proper directory of your file tree
 
-1. Using the Terminal, ensure you are in the `recipe-page` directory. Create a file called `styles.css`. You can verify this worked by running the `ls` command.
-2. Use the Terminal shortcut `code .` to open the `recipe-page` directory in VSCode.
-3. Run `git status`. You should see that 'styles.css' is not being tracked yet.
-4. Add and commit, making sure to write a descriptive commit message in the present tense, such as "Create stylesheet."
+1. On your desktop, create a folder called `techtonica-projects`.  Move your `recipe-page.html` file into `techtonica-projects`.  If you no longer have it, make it again with the Challenge directions from [Phase 1](./phase-1-html-prompt.md).
```
Comment (SarahCawley) [inline projects/recipe-page/phase-2-css-prompt.md:69]: They already created the folder in part 1, so maybe add the "if you have not already created the folder" part to this

Comment (SarahCawley) [reply projects/recipe-page/phase-2-css-prompt.md:69]: The first part was done on replit, I am not sure why we are moving it locally...

File: web/css-part-2.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+### Materials
+[CSS Diner](https://flukeout.github.io/)
+[Flexbox Froggy](https://flexboxfroggy.com/)
+[Flexbox Cheatsheet](https://flexbox.malven.co/)
+[Add CSS to recipe-page project](../projects/recipe-page/phase-2-css-prompt.md)
+[Article: Inline CSS](https://www.codecademy.com/articles/html-inline-styles)
+
+## Lesson
+
+CSS concepts are not hard; it's using them all together effectively and neatly on a large scale that challenges people.  Experience is key, so let's get you some.
```
Comment (SarahCawley) [inline web/css-part-2.md:45]: Can we add something like: You are not expected to have this all memorized at the end of the lesson. Right now just get used to looking at CSS and try to see how the CSS relates to the HTML.

Is that too simplistic? I just worry people will go down rabbit holes and feel overwhelmed

File: web/css-part-2.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+The activity for the HTML, CSS, and JS DOM lessons is a project that allows you to combine your new skills in each of these areas.
+
+[Click here to access the instructions for the CSS portion of the recipe-page project](../projects/recipe-page/phase-2-css-prompt.md). Please don't move on to the prompts for the other topics yet.
+
+Check for [Inline CSS](https://www.codecademy.com/articles/html-inline-styles) contents while removing _style_ tag from the webpage.
+
+Tutorials for changing and removing style tags are provided in the [CSS slides](https://docs.google.com/presentation/d/1p-IXWxo0NEbZbHQ_Mdoo-A9dlFXqfSPOfW6navfyeTI/edit?usp=sharing)
+
+### Challenge
+
```
Comment (SarahCawley) [inline web/css-part-2.md:79]: If there is no challenge should we remove this section?

**Review by SarahCawley — COMMENTED** [review]
Looks great overall! Thanks for removing the bootstrap stuff! I had just a couple of things I wanted to clarify

File: projects/recipe-page/phase-2-css-prompt.md (hunk 1 above)
Comment (alodahl) [reply projects/recipe-page/phase-2-css-prompt.md:74]: Yes, you wanted this to happen before they learn CLI, correct?  CLI is a prereq of VSCode.
https://docs.google.com/spreadsheets/d/1xPgOLI4nEOih3Y5QzpVMLNgDX9g5xjzeE7wfjirWA0M/edit#gid=0

File: projects/recipe-page/phase-2-css-prompt.md (hunk 2 above)
Comment (alodahl) [reply projects/recipe-page/phase-2-css-prompt.md:69]: In the HTML part, we built it on CodePen to start out simple, and then moved it to local at the end.  I figured they would miss out on connecting things through style and script tags if we didn't move to local, but I didn't want  them to get frustrated before writing their first HTML tags with setup overhead. Let me know if you think I should switch to just one or the other.

File: web/css-part-2.md (hunk 3 above)
Comment (alodahl) [reply web/css-part-2.md:45]: I like it

File: web/css-part-2.md (hunk 4 above)
Comment (alodahl) [reply web/css-part-2.md:79]: good catch

File: web/css-part-2.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  - [::before pseudo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)
+1. Although CSS issues tend to be relatively easy to Google, https://css-tricks.com/ has one of the best collections of **CSS Guides**.  Take 15 minutes total to look at some of the most popular articles. You don't need to read them throughly, but instead get a feel for some of the common problems you'll be solving.
+  - Flexbox: https://css-tricks.com/snippets/css/a-guide-to-flexbox/
+  - Centering: https://css-tricks.com/centering-css-complete-guide/
+  - Color: https://css-tricks.com/nerds-guide-color-web/
+  - Media Queries: https://css-tricks.com/a-complete-guide-to-css-media-queries/
+  - Links and Buttons: https://css-tricks.com/a-complete-guide-to-links-and-buttons/
+  - Position Property (video): https://css-tricks.com/video-screencasts/198-about-the-position-property/
+
+### Independent Practice
```
Comment (alodahl) [inline web/css-part-2.md:66]: note to self:  separate this too?

File: projects/recipe-page/phase-2-css-prompt.md (hunk 1 above)
Comment (alodahl) [reply projects/recipe-page/phase-2-css-prompt.md:65]: I removed Git for the same reason.

File: projects/recipe-page/phase-2-css-prompt.md (hunk 2 above)
Comment (alodahl) [reply projects/recipe-page/phase-2-css-prompt.md]: I've made edit's so that this is the first time we're asking them to make these directories.

**Comment by alodahl** [comment]
I think this set of changes is a finished unit now.  Let me know what you would like adjusted @SarahCawley.

File: projects/recipe-page/phase-1-html-prompt.md [hunk 6]
```diff
@@ -200,12 +173,19 @@ You should see changes that need to be added and committed. Add and commit them,
...
 
-### Extensions
+### Challenge
+See the HTML in your browser by following just the steps under the sections "Part 1: Saving an HTML File" and "Part 2: Running an HTML File" in https://www.wikihow.com/Run-a-HTML-File. You should name your file `recipe-page.html`. If you are on a Windows PC, use WordPad instead of TextEdit. (10 min)
+
+Copy everything from your HTML in CodePen, paste it into `recipe-page.html`, and save. Follow the directions in the link to view it in your browser.
+
+Did it work?  If not, show it to a peer and figure out what you're missing.
+
+### Extensions (optional)
```
Comment (SarahCawley) [inline projects/recipe-page/phase-1-html-prompt.md:188]: Perfect!

**Review by SarahCawley — APPROVED** [review]
Looks great! Thanks for adding the explanations on why we are switching to the VS code from the codepen.

**Pull request merged**